src/Gui/RemoteDebugger.py

```diff
@@ -25,6 +25,7 @@
 import FreeCAD as App
 import FreeCADGui as Gui
 from PySide import QtGui
+from freecad.utils import get_python_exe
```

Comment:
Why is this not `FreeCAD` ?  See https://github.com/FreeCAD/FreeCAD/pull/9663#issuecomment-1622583028